docs/linux/reporting_kernel_bugs.md

```diff
@@ -1,11 +1,39 @@
 # Reporting Linux kernel bugs
 
-Before reporting a bug make sure nobody else already reported it. The easiest way to do this is to search through the [syzkaller mailing list](https://groups.google.com/forum/#!forum/syzkaller), [syzkaller-bugs mailing list](https://groups.google.com/forum/#!forum/syzkaller-bugs) and [syzbot dashboard](https://syzkaller.appspot.com/upstream) for key frames present in the kernel stack traces.
-
-Please report found bugs to the Linux kernel maintainers.
+Before reporting a bug **make sure nobody else has already reported it**. The easiest way to do
+this is to search through the [syzkaller mailing list](https://groups.google.com/forum/#!forum/syzkaller),
+[syzkaller-bugs mailing list](https://groups.google.com/forum/#!forum/syzkaller-bugs),
+[syzbot dashboard](https://syzkaller.appspot.com/upstream),
+and [kernel mailing lists](https://lore.kernel.org/) for key frames present in the kernel stack traces.
+
+**Do NOT report bugs on old kernel versions**.
+Old kernel generally means older than a week. Reports on old kernels are usually negatively accepted,
+and asked to be reproduced on a fresh kernel. Do not report bugs found on stable/LTS kernels.
+Bug on stable/LTS kernels should be reproduced on a fresh upstream kernel, and reported as such.
+Or, if they are already fixed upstream, the fixing patch should be
```

Comment:
s/fixing patch/fix commit/

assuming we're only talking about already commited fixes.

Reply:
Done